### ISSUE TITLE
Installation not done!

### ISSUE THREAD

**Issue by p30rayan** [cloverstudio/Spika-Server#93]
Hi i want run server side in xampp in localhost. I install composer and i config installer and installer show perfect. But after putting db info and creating database tables, the installer not going to final step and show me error and says: installation faild! Try again. I tried many times but problem still happens

**Comment by snetts** [comment]
Hello @p30rayan, I am struggling to install Spika-Server on my local Wamp environment. Could you please help me coz I am stuck. Thanks.

**Comment by p30rayan** [comment]
hi dear friend. for install u must use vps or ded server i think

On Mon, Jan 18, 2016 at 11:17 AM, Victor Okech notifications@github.com
wrote:

> Hello @p30rayan https://github.com/p30rayan, I am struggling to install
> Spika-Server on my local Wamp environment. Could you please help me coz I
> am stuck. Thanks.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cloverstudio/Spika-Server/issues/93#issuecomment-172453600
> .

## 

برنامه نویس و طراح حرفه ای وب و اپلیکیشن موبایل و ویندوز با ۷ سال سابقه کاری

Web, mobile app and windows app Developer & Designer with 7 years experience

**Comment by snetts** [comment]
Yeah, I presume. I am trying to setup a development, staging, production environment. I have a dedicated server where the live application is running but I need to have a local development environment because I don't want to be making changes to the production server and messing things up. I don't want to be a "Cowboy Coder", you understand. That is why I am trying to install on Wamp.

I have managed to get to the installation page. Let me see how far it goes. I will update you of my progress.

Thanks @p30rayan

**Comment by p30rayan** [comment]
I'll try to do this on my system. if it's worked, then i let you know with
all details.

On Mon, Jan 18, 2016 at 11:30 AM, Victor Okech notifications@github.com
wrote:

> Yeah, I presume. I am trying to setup a development, staging, production
> environment. I have a dedicated server where the live application is
> running but I need to have a local development environment because I don't
> want to be making changes to the production server and messing things up. I
> don't want to be a "Cowboy Coder", you understand. That is why I am trying
> to install on Wamp.
> 
> I have managed to get to the installation page. Let me see how far it
> goes. I will update you of my progress.
> 
> Thanks @p30rayan https://github.com/p30rayan
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cloverstudio/Spika-Server/issues/93#issuecomment-172456255
> .

## 

برنامه نویس و طراح حرفه ای وب و اپلیکیشن موبایل و ویندوز با ۷ سال سابقه کاری

Web, mobile app and windows app Developer & Designer with 7 years experience

**Comment by snetts** [comment]
@p30rayan Progress update, I have managed to go through and complete the installation process, but I can't access the admin back-end.

**Comment by p30rayan** [comment]
so whats the error? if u can send me scrn shut

On Mon, Jan 18, 2016 at 1:23 PM, Victor Okech notifications@github.com
wrote:

> @p30rayan https://github.com/p30rayan Progress update, I have managed
> to go through and complete the installation process, but I can't access the
> admin back-end.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/cloverstudio/Spika-Server/issues/93#issuecomment-172481983
> .

## 

برنامه نویس و طراح حرفه ای وب و اپلیکیشن موبایل و ویندوز با ۷ سال سابقه کاری

Web, mobile app and windows app Developer & Designer with 7 years experience

**Comment by snetts** [comment]
Okay there is a step that does not complete well when doing the installation. Let me try again and I will update you.

**Comment by snetts** [comment]
So here is my progress. I followed all the instructions in the install file that comes with the Spika-Server repository and I managed to get a local install of Spika working. Funny thing though I couldn't get it to work with the github repo, so I compressed the files on my server and downloaded them to my local server, extracted and proceeded with the installation and funnily enough the installation process completed successfully. If you need any help on the process let me know. Thanks @p30rayan
